src/main/java/net/minecraftforge/common/util/ModFixs.java

```diff
@@ -31,24 +31,59 @@
 import net.minecraft.util.datafix.IFixType;
 import net.minecraft.util.datafix.IFixableData;
 
+/**
```

Comment:
Remove these. `ModFixs` are too complicated to have generic javadocs.

Reply:
Fixed

Reply:
> Remove these. `ModFixs` are too complicated to have generic javadocs.

I don't think so, I think there should be a java doc telling modders how to register and get ModFix.

Reply:
Yes, but not with AI, it has to be written with detail.